lib/generators/blacklight/install_generator.rb

```diff
@@ -88,7 +88,7 @@ def generate_blacklight_marc_demo
       return unless options[:marc]
 
       blacklight_marc = String.new('blacklight-marc')
-      gem blacklight_marc, '~> 6.1'
+      gem blacklight_marc, '= 7.0.0.rc1'
```

Comment:
should this be `>= 7.0.0.a`?

Reply:
Like "pre-release or greater of this dot release," right? Looking around on the internet, I can't seem to find documentation on how to specify this, the closest thing I could find was https://github.com/bundler/bundler/issues/1198#issuecomment-1228741.  These seem like they're all possible, good options?

`= 7.0.0.rc1` (I know this one works)
`>= 7.0.0.a`
`>= 7.0.0.beta`
`~> 7.0.0.a`
`~> 7.0.0.beta`

Reply:
Okay, how about ` < 8`?  I'm thinking we don't want to have to revisit this line once we release newer versions of blacklight-marc.